src/Javis.jl

```diff
@@ -1225,7 +1238,7 @@ end
 Get one specific frame of a video with actions.
 
 # Returns
-- `ARGB Matrix` the frame image as a matrix
+- `ARGB Matrix` - the frame image as a matrix
```

Comment:
Maybe use `typeof` to get the actual type or at least specify that it is two dimensional to be more exact?

Reply:
This is updated and fixed.

Reply:
No it's still exactly the same :stuck_out_tongue:

Reply:
Fixed now.